tests/test_C1_verification.py

```diff
@@ -0,0 +1,28 @@
+import os
+import pytest
+
+from .constants import REPO_PATH
+import src.C1_verification as C1
+
+
+def test_lookup_file_existing_file():
+    file_name = os.path.join(os.path.dirname(__file__), "inputs", "mvs_config.json")
+    # no error is risen
+    C1.lookup_file(file_path=file_name, name="test")
```

Comment:
How is this asserting, whether or not an error is raised?

Reply:
A test always fails if an error is risen except you tell pytest to expect the error (with pytest.raises ...).

Reply:
I would make use of the constant variables for "inputs" and "mvs_config.json", that way, if they change, the test remains valid

Reply:
Thanks @Bachibouzouk  I'm using constants now. The tests/constants.py though.

Reply:
Why not the `src/constants.py` :) ?

Reply:
Because I am using the mvs_config.json in the `tests` folder. Also in tests_D1.py I will use folders and files of the test folder.

Reply:
Ok, but then I would import the values form the `src/constants.py` within the `tests/constants.py` so we avoid duplicates

Reply:
okay!